### PR TITLE
[15.0][MIG] product_secondary_unit

### PR DESCRIPTION
**Pull request by remihb**
As title says.

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]

**Review by BT-aestebanez — APPROVED** [review]
LGTM

**Comment by BT-aestebanez** [comment]
@rousseldenis This is already reviewed and approved, maybe you can merge it and update the entry in the issue https://github.com/OCA/product-attribute/issues/919 (it is duplicated) ?
Thanks

**Comment by BT-aestebanez** [comment]
@sergio-teruel This is already reviewed and approved, maybe you can merge it and update the entry in the issue https://github.com/OCA/product-attribute/issues/919 (it is duplicated) ?
Thanks

**Comment by BT-aestebanez** [comment]
@rousseldenis Could you please merge it?

File: product_secondary_unit/tests/test_secondary_unit_mixin.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(cls):
         cls.loader.restore_registry()
-        super(TestProductSecondaryUnitMixin, cls).tearDownClass()
+        return super(TestProductSecondaryUnitMixin, cls).tearDownClass()
```
Comment (rousseldenis) [inline product_secondary_unit/tests/test_secondary_unit_mixin.py:63]: Mmmh, not sure return is mandatory

File: product_secondary_unit/README.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github
+    :target: https://github.com/OCA/product-attribute/tree/14.0/product_secondary_unit
+    :alt: OCA/product-attribute
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_secondary_unit
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/135/14.0
```
Comment (sergio-teruel) [inline product_secondary_unit/README.rst]: Please re-genrate this file

File: product_secondary_unit/tests/test_secondary_unit_mixin.py (hunk 1 above)
Comment (BT-aestebanez) [reply product_secondary_unit/tests/test_secondary_unit_mixin.py:63]: I don't think so:
```
@classmethod
def tearDownClass(cls):
      "Hook method for deconstructing the class fixture after running all tests in the class."
```

This is the definition on python 3.7.

Comment (BT-aestebanez) [reply product_secondary_unit/tests/test_secondary_unit_mixin.py:63]: @remihb could you remove this return so the module can be merged?
Thanks,

@rousseldenis FYI (thanks for the review :) )

**Comment by sergio-teruel** [comment]
Plase @BT-aestebanez Can you add this commit in v14 and v15??
https://github.com/OCA/product-attribute/pull/993/commits/f68840f74415519ca5846973045762539609442b

File: product_secondary_unit/tests/test_secondary_unit_mixin.py (hunk 1 above)
Comment (remihb) [reply product_secondary_unit/tests/test_secondary_unit_mixin.py:63]: Done.
But pre-commit fails without return

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by BT-aestebanez** [comment]
@sergio-teruel thanks for the review, but I am not the author.
@remihb Can you do what @sergio-teruel suggest?

Thanks.

File: product_secondary_unit/tests/test_secondary_unit_mixin.py (hunk 1 above)
Comment (BT-aestebanez) [reply product_secondary_unit/tests/test_secondary_unit_mixin.py:63]: True
`product_secondary_unit/tests/test_secondary_unit_mixin.py:61: [W8110(missing-return), TestProductSecondaryUnitMixin.tearDownClass] Missing `return` (`super` is used) in method tearDownClass.`

What do you suggest @rousseldenis ?

I'd keep the return

**Comment by sergio-teruel** [comment]
Sorry @BT-aestebanez :smile:

**Comment by BT-aestebanez** [comment]
@remihb Can you apply the suggested changes? otherwise I can create a new PR to do it and close this :)

**Comment by remihb** [comment]
> @remihb Can you apply the suggested changes? otherwise I can create a new PR to do it and close this :)

Sorry I missed that. Ok now it's done.

**Comment by BT-aestebanez** [comment]
> > @remihb Can you apply the suggested changes? otherwise I can create a new PR to do it and close this :)
> 
> Sorry I missed that. Ok now it's done.

You forgot to add the commit mentioned: https://github.com/OCA/product-attribute/commit/f68840f74415519ca5846973045762539609442b

**Comment by remihb** [comment]
> > > @remihb Can you apply the suggested changes? otherwise I can create a new PR to do it and close this :)
> > 
> > 
> > Sorry I missed that. Ok now it's done.
> 
> You forgot to add the commit mentioned: [f68840f](https://github.com/OCA/product-attribute/commit/f68840f74415519ca5846973045762539609442b)

Alright, done.

**Comment by BT-aestebanez** [comment]
> > > > @remihb Can you apply the suggested changes? otherwise I can create a new PR to do it and close this :)
> > > 
> > > 
> > > Sorry I missed that. Ok now it's done.
> > 
> > 
> > You forgot to add the commit mentioned: [f68840f](https://github.com/OCA/product-attribute/commit/f68840f74415519ca5846973045762539609442b)
> 
> Alright, done.

Thanks @remihb 

@sergio-teruel I think this can already be merged, do you?

**Comment by BT-aestebanez** [comment]
@sergio-teruel ping

**Comment by sergio-teruel** [comment]
Please fix pre-commit. Thanks

**Comment by BT-aestebanez** [comment]
@remihb Can you add again the "return"? It seems this is coming to an end hehe

**Comment by remihb** [comment]
> @remihb Can you add again the "return"? It seems this is coming to an end hehe

Done

**Comment by BT-aestebanez** [comment]
@sergio-teruel could you merge it please?

**Comment by BT-aestebanez** [comment]
@sergio-teruel ping

**Review by CarlosRoca13 — APPROVED** [review]
Code review

**Comment by sergio-teruel** [comment]
@BT-aestebanez Sorry but I do not have merge permission on this repository

**Pull request merged**